src/varity/vcf_to_hgvs/protein.clj

```diff
@@ -231,13 +253,14 @@
         [_ _ offset _] (diff-bases pref palt)
         ref (nth (:ref-prot-seq seq-info) (dec (+ ppos offset)))
         alt (nth (:alt-prot-seq seq-info) (dec (+ ppos offset)))
-        ter-site (-> (:alt-prot-seq seq-info)
+        ter-site (-> seq-info
+                     format-alt-prot-seq
                      (subs (dec (+ ppos offset)))
                      (string/index-of "*"))]
     (mut/protein-frame-shift (mut/->long-amino-acid ref)
                              (coord/protein-coordinate (+ ppos offset))
                              (mut/->long-amino-acid alt)
-                             (if ter-site
+                             (if (and ter-site (< 1 ter-site))
```

Comment:
You are right, and I understand it. `fs*1` is not frame-shift, but `(< 1 ter-site)` here also excludes `fs*2`.